release-notes/Cloud/DCP_change_log.md

```diff
@@ -10,6 +10,10 @@ The dataminer.services platform gets updated continuously. This change log can h
 
 In the Admin app, it is now possible to export audit records to a CSV file. You can configure the separator to use in the export and select whether column titles should be included as the header in the file.
 
+#### 30 May 2023 - Enhancement - Catalog 1.3.1 - Notification when deploying from the catalog [ID_36543]
```

Comment:
I would remove the version from next to "Catalog" since we do not apply versioning towards the users for our UI apps.